rpxy-bin/src/config/toml.rs

```diff
@@ -107,6 +108,8 @@ impl TryInto<ProxyConfig> for &ConfigToml {
       // listen port and socket
       http_port: self.listen_port,
       https_port: self.listen_port_tls,
+      https_redirection_port: if self.tls_redirection_port.is_some() {
```

Comment:
Here we have to update the struct type of `ProxyConfig` including `https_redirection_port`.